website/app/page.tsx

```diff
@@ -26,7 +27,7 @@ const Home = () => {
           Welcome UI is a customizable design system library made with react, typescript,
           styled-components, ariakit and a lot of love 💛
         </Text>
-        <Button as={NextLink} href="/foundations/getting-started" mt="3xl" size="lg">
+        <Button as={NextLink} className="!mt-3xl" href="/foundations/getting-started" size="lg">
```

Comment:
I know this not working without `!` but we need to fix after button instead of add important

```suggestion
        <Button as={NextLink} className="mt-3xl" href="/foundations/getting-started" size="lg">
```